content/guide/_index.md

```diff
@@ -8,145 +8,207 @@ Welcome! This guide will help you get started with a Hugo Theme, including how t
 
- * `title` is for your site title. Which shown on the browser tab.
+The next time you run Hugo it will download the theme. You will not see that theme in your `themes` folder, because Hugo caches it in a temporary directory. But no need to worry, it will find the theme files.
 
- * `theme` name is already provided with every theme. don't need to customize this theme name.
+You will find a file called `go.sum` in your root directory. Add this file to your repository.
 
- * `summaryLeangth` is for the post excerpt limit. you can set your post excerpt length as you want to show.
+### Step 5: Copy the sample site
 
- * `paginate` set your pagination for page items. Such as blog posts, portfolio items, etc. If you set `paginate = 10` then it will show 10 posts on a page.
+Every theme comes with a folder named `exampleSite`. Copy the folders and files inside of that folder into your root directory. Be careful not to overwrite anything you already changed, but expect to override some files and folders that were created by step 2 above.
 
- * `googleAnalytics` is for activating analytics on your site. If you want google analytics to activate, then simply give your analytics ID here (i.e. `googleAnalytics= UA-123-45`). Or if you don't, then leave it empty.
+### Step 6: Add files to the repository
 
- * `disqusShortname` is for activating comment section on a blog page or other page which needs a comment section. get your Disqus short name from [here](https://disqus.com/). if you don't need the comment section, then leave it empty.
+Now your site is ready to run. Add a file with the name `.gitignore` to your root directory and fill it with the contents of [this autogenerated sample file](https://www.toptal.com/developers/gitignore/api/node,go,hugo). This will exempt several files from being added to the repository that are created automatically. You can change the list of supported programming languages and IDEs [at this link](https://www.toptal.com/developers/gitignore?templates=node,go,hugo).
 
- * `DefaultContentLanguage` field is for your site default language, which will load first with your site load.
+After that add all files to your repository and commit.
 
- * `disableLanguages` field is for disabling languages, if you don't want to show one or more then one language then you can add the language name here, it will disable those languages from your site. example `disableLanguages = ["fr"]` it will disable french language from your site.
+```bash
+git add -A
+git commit -m "first commit of all website files"
+```
 
-#### Default Parameters
+### Step 7: Test your website
 
-Default parameters are those defined by the theme developer. Default parameters start with `[params]` heading.
+Hugo comes with an integrated development server. You can easily start the server with the following command:
```

Comment:
Codacy found an issue: [[maximum-line-length] Line must be at most 80 characters](https://app.codacy.com/gh/gethugothemes/gethugothemes-docs/pullRequest?prid=6304190)